### PR TITLE
Remove support for QCOW2

### PR DESCRIPTION
**Pull request by gthvn1**
The following PR [xcp-ng-xapi-storage#22](https://github.com/xcp-ng/xcp-ng-xapi-storage/pull/22) is required in order to work.

### CODE REVIEW

**Comment by stormi** [comment]
This should probably be a draft until the code PR is merged?

File: SPECS/xcp-ng-xapi-storage.spec [hunk 1]
```diff
@@ -79,6 +78,8 @@ cd build
 %{_docdir}/xcp-ng-xapi-storage/
 %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/
 %{_prefix}/lib/python2.7/site-packages/xcp_ng_xapi_storage_libs-*-py2.7.egg-info
+%exclude %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/libcow/qcow2util.py
+%exclude %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py
```
Comment (stormi) [inline SPECS/xcp-ng-xapi-storage.spec:83]: I'd add a comment explaining why (I know it's already in the changelog, but it saves time when it's right in front of your eyes)

**Review by ydirson — APPROVED** [review]

**Review by stormi — APPROVED** [review]

**Comment by gthvn1** [comment]
koji scratch build: http://koji.xcp-ng.org/taskinfo?taskID=57544

**Comment by stormi** [comment]
Is it early for a scratch build? I don't see the tarball updated in the PR.

**Comment by gthvn1** [comment]
Yes too early. I thought that tarball will be downloaded using "Source0"...

**Comment by gthvn1** [comment]
scratch build shoud work better now:  http://koji.xcp-ng.org/taskinfo?taskID=57587

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Koji build to v8.3-incoming: http://koji.xcp-ng.org/taskinfo?taskID=58246

**Pull request merged**